sympy/stats/symbolic_probability.py

```diff
@@ -109,17 +125,19 @@ def __new__(cls, expr, condition=None, **kwargs):
             condition = _sympify(condition)
             obj = Expr.__new__(cls, expr, condition)
         obj._condition = condition
+        obj._evaluate = kwargs.get('evaluate', True)
```

Comment:
There is a global variable defining the default behaviour for the evaluation. Can't recall it right now, but somewhere in `sympy/core` it's defined.